index.php

```diff
@@ -110,6 +126,8 @@ public function register_custom_menu_page() {
 		public function bsf_settings_link( $links ) {
 			$settings_link = '<a href="admin.php?page=rich_snippet_dashboard">Settings</a>';
 			array_unshift( $links, $settings_link );
+			$pro_link = '<a href="https://wpschema.com/pricing/?utm_source=Schema&utm_medium=Plugin-list&utm_campaign=upgrade" target="_blank" style="color: #d54e21; font-weight: bold;">Get Schema Pro</a>';
```

Comment:
All these pro links should not be added if schema pro is already installed or active on site, are we checking this?